contracts/Math.sol

```diff
@@ -137,7 +137,7 @@ library GnosisMath {
 
     /// @dev Returns base 2 logarithm value of given x
     /// @param x x
-    /// @return logarithmic value
+    /// @return lo logarithmic value
```

Comment:
nit: remove space.

Reply:
I don't see a space here on mobile. Will check it on the computer in the morning!

Reply:
Looks correct to me as well. I think this was required becuase of NatSpec enforcement in the doc comments by the compiler (<https://docs.soliditylang.org/en/v0.6.12/natspec-format.html>). The `lo` is the name of the return variable that needs to be included in the doc comment (<https://docs.soliditylang.org/en/v0.6.12/natspec-format.html?highlight=return#tags>).

Reply:
I see. I wasn't expecting changes in the comments and to me it looked like a typo to have `lo logarithmic`. All good then.

Reply:
I could put a little dash here. Alternatively I would like to rename the return var to log. It would then be `@returns log - logarithmic value`